app/controllers/api/v1/articles_controller.rb

```diff
@@ -1,13 +1,18 @@
 class Api::V1::ArticlesController < ApplicationController
   rescue_from ActiveRecord::RecordNotFound, with: :not_found
 
+  def index
+    articles = Article.paginate(page: params[:page], per_page: 4)
+    render json: articles, each_serializer: Articles::IndexSerializer, meta: meta_attributes(articles)
+  end
+
   def show
     article = Article.find(params[:id])
     render json: article, serializer: Articles::ShowSerializer
   end
 
   private
-  
+
   def not_found
```

Comment:
I would now move this one to the application controller and change the error message to `Article not found`

Reply:
Done